### ISSUE TITLE
转换后的模型预测不准确，原因是float类型

### ISSUE THREAD

**Issue by OKC13** [lany192/MnistTFLite#1]
python中，tf.keras.utils.normalize得到的是float64

而TFlite默认是float32。

**Comment by OKC13** [comment]
除此之外，可能java中传递的格式不一致。

**Comment by OKC13** [comment]
因为训练时对输入的int做了归一化处理，而java中未做处理。
